src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+
+  const x = sourceString
+    .toString()
+    .split(';')
+    .map((item) => item.toString().split(':'));
```

Comment:
[CODE STYLE] - All chained methods should have the same indentation to improve readability.